packages/ui/components/select-rich/src/LionSelectRich.js

```diff
@@ -526,7 +526,7 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
   _listboxOnKeyDown(ev) {
     super._listboxOnKeyDown(ev);
 
-    if (this.disabled) {
+    if (this.disabled || this.readOnly) {
```

Comment:
This part of the code will only be triggered when the focus is on the listbox, so when already opened. It doesn't make sense to check if `readOnly` is on since the listbox should not get opened in the first place.

I'm not sure why we check for disabled here, i would expect that is also not needed.

Reply:
Yes, makes sense! reverted this part